modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchIndexSearcher.java

```diff
@@ -345,6 +323,13 @@ protected void setQuery(
 		baseSearchRequest.setQuery(searchRequest.getQuery());
 	}
 
+	private void _closePointInTime(PointInTime pointInTime) {
```

Comment:
inline in the caller:
_searchEngineAdapter.execute(new ClosePointInTimeRequest(pointInTime.getPointInTimeId());